packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -46,13 +46,11 @@ export function getSPOAndGraphRequestIdsFromResponse(headers: { get: (id: string
         { headerName: "sprequestguid", logName: "sprequestguid" },
         { headerName: "request-id", logName: "requestId" },
         { headerName: "client-request-id", logName: "clientRequestId" },
-        { headerName: "x-msedge-ref", logName: "xMsedgeRef" },
```

Comment:
This property is for SPO calls. Does SPO has AFD vs. non-AFD paths? If yes, I assume we want to keep it.